### DISCUSSION TITLE
How to run multiple commands in one `run=` of keymap.toml?

### DISCUSSION THREAD

**Discussion by c02y** [sxyazi/yazi#898] [1. Q&A] [answered]
Sometimes I need to run multiple commands in one `run=` of keymap.toml such as:
when I try to sort the files/dirs list according to mtime or size, I need to show the mtime or size of all the files/dirs of the list at the same time.

Is it able to do that right now?

**Comment by sxyazi** [comment, marked as answer]
`run` can accept an array if you want to run multiple commands:

```
run = ["command1", "command2"]
```

**Reply by Emptyfruit** [reply]
Should it work if the second command is "quit"? It seems like it does not..
